packages/design-system/src/__mocks__/i18next.js

```diff
@@ -7,5 +7,9 @@ i18next.init = jest.fn();
 i18next.addResourceBundle = jest.fn();
 i18next.changeLanguage = jest.fn();
 i18next.language = 'en';
+i18next.t = function (key, params) {
+  const paramString = params ? ` | ${JSON.stringify(params)}` : '';
+  return `${key}${paramString}`;
+};
```

Comment:
Moved the existing translation mock to the `i18next` file instead of the `react-i18next` file so it can be used outside of react